### PR TITLE
ushift: Skip cluster operator monitoring

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by ingvagabund** [comment]
s/Remove cluster operator monitoring/Skip cluster operator monitoring initialization/ or similar is more suitable for the summary. The changes in the PR are not actually removing anything.

**Comment by pacevedom** [comment]
> s/Remove cluster operator monitoring/Skip cluster operator monitoring initialization/ or similar is more suitable for the summary. The changes in the PR are not actually removing anything.

Done

File: pkg/monitor/operator.go [hunk 1]
```diff
@@ -11,12 +11,30 @@ import (
...
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 
 	configv1 "github.com/openshift/api/config/v1"
 	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 )
 
+func isClusterOperatorPresent(client kubernetes.Interface) (bool, error) {
```
Comment (ingvagabund) [inline pkg/monitor/operator.go:21]: Any chance moving `DoesApiResourceExist` some place else where it can be used in both the tests and in here?

**Pull request closed without merging**